frontend/src/lib/utils.js

```diff
@@ -1,33 +1,52 @@
 export const getHouseColor = (house) => {
-  switch (house) {
-    case "Gryffindor":
-      return "#ae0001";
-    case "Slytherin":
-      return "#2a623d";
-    case "Hufflepuff":
-      return "#ffdb00";
-    case "Ravenclaw":
-      return "#222f5b";
-    default:
-      return "#bebebe";
-  }
+	switch (house) {
+		case 'Gryffindor':
+			return '#ae0001';
+		case 'Slytherin':
+			return '#2a623d';
+		case 'Hufflepuff':
+			return '#ffdb00';
+		case 'Ravenclaw':
+			return '#222f5b';
+		default:
+			return '#bebebe';
+	}
+};
+
+export const getPotionDifficultyColor = (difficulty) => {
+	switch (difficulty) {
+		case 'Beginner':
+			return 'white';
+		case 'Ordinary Wizarding Level':
+			return '#bebebe';
+		case 'Moderate':
+			return '#bdae2d';
+		case 'Moderate to Advanced' || 'Moderate/Advanced':
+			return '#dbae3b';
+		case 'Advanced':
+			return '#9e1919';
+		case 'One of a kind':
+			return '#21a376';
+		default:
+			return 'transparent';
+	}
 };
```

Comment:
Setting the potion color by difficulty is a good idea! 
But I am not quite sure about the color palette. It looks like [this ](https://coolors.co/ffffff-bebebe-bdae2d-dbae3b-9e1919-21a376 ) basically, doesn't it? I would maybe prefer a palette from light green (beginner) to dark red (advanced). 
And I would use `#includes(level)` as it there are like 12 different variants of difficulty levels (advanced, advanced to beginner, beginner, beginner to advanced, beginner to moderate, beginner to ordinary wizarding level, moderate to advanced, moderate to ordinary wizarding level, moderate/advanced, one of a kind and ordinary wizarding level). 
That would maybe cover all variants.

Reply:
What I meant with that is using a the [`include`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) method to check if the difficulty contains one of the difficulty levels. 

like: 
``` 
if difficulty.includes("beginner") {
  return "#fff"
} else difficulty.includes("advanced") {
 return "#bebebe"
} 
...
```
Maybe we can also combine it with a regex or switch, but that'S how I would do it basically.

Reply:
I see now, I'll try something

Reply:
I put this color palette: https://coolors.co/8ee485-7dc44f-45a85e-d8c624-db6e3b-801c1c

Reply:
That's a nice palette!